packages/bitcore-node/package.json

```diff
@@ -17,7 +17,7 @@
     "printWidth": 120
   },
   "scripts": {
-    "start": "npm run tsc && node build/src/server.js",
+    "start": "npm run tsc && node --max-old-space-size=8192 build/src/server.js",
```

Comment:
So I've been testing with an `r5.large` and it seems to be enough, but that's probably the case since the node didn't fully use the `8GB` (since its just an upper bound), but I'll see if there are other solutions that don't involve increasing the heap upper limit.